src/encoded/tests/test_audit_biosample.py

```diff
@@ -1,6 +1,5 @@
 import pytest
 
-
 def test_audit_biosample_modifications_whole_organism(
```

Comment:
please - if you haven't modified/touched the file return it to the way it has been before.